node/test/unit/openshift_test.rb

```diff
@@ -159,7 +159,6 @@ def test_cartridge_do_action_quota
     @agent.expects(:request).returns(request).times(3)
 
     @agent.expects(:execute_action).with('test_agent', any_parameters).returns [0, 'test output', nil]
-    @agent.expects(:report_quota).with(instance_of(String), 'uuid')
```

Comment:
Could you explain why this was removed? It looks like the test uses the action 'test_agent', which should match your conditional and report quota.